modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/baseComponents/BaseAPIEndpointFields.tsx

```diff
@@ -80,8 +80,8 @@ export default function BaseAPIEndpointFields({
 	);
 
 	const endpointPathHostTextPreview =
-		selectedScope?.value === 'group'
-			? `${window.location.origin}${basePath}${apiApplicationBaseURL}/scopes/${selectedScope.value}`
+		selectedScope?.value === 'site'
+			? `${window.location.origin}${basePath}${apiApplicationBaseURL}/scopes/{scopeKey}`
```

Comment:
I think the update to {scopeKey} is done in purpose, but I am writing this comment just to be sure

Reply:
Yeah, this was a bug that was shown in the UI